cql/DashboardLibrary.cql

```diff
@@ -162,6 +167,20 @@ define function CytologyInterpretation(conclusions List<System.Concept>):
         when c ~ "ASC-H" then {text: 'ASC-H', rank: 5}
         when c ~ "AGC" then {text: 'AGC', rank: 6}
         when c in "HSIL" or c ~ "SCC" then {text: 'HSIL+', rank: 7}
+        when c ~ "AGC Favor Neoplasia" then {text: 'AGC-FN', rank: 8}
+        when c ~ "Endocervical Cells Favor Neoplasia" then {text: 'AEC-FN', rank: 8}        
+        when c in "AIS" then {text: 'AIS', rank: 9}
+
+        // Rare cytologies added so they are not interpreted as UNK, but ranked low, to-do: confirm names
+        when c ~ "Endocervical AIS" then {text: 'Endocervical AIS', rank: 1}
+        when c ~ "Histiocytes" then {text: 'Histiocytes', rank: 1}
+        when c ~ "Atypical Endocervical Cells" then {text: 'Atypical Endocervical Cells', rank: 1}
+        when c ~ "Atypical Endometrial Cells" then {text: 'Atypical Endometrial Cells', rank: 1}
+        when c ~ "Benign Endometrial Cells" then {text: 'Benign Endometrial Cells', rank:1}
+        when c ~ "Unsatisfactory" then {text: "Unsatisfactory", rank: 1}
```

Comment:
Single quotes

```suggestion
        when c ~ "Unsatisfactory" then {text: 'Unsatisfactory', rank: 1}
```

Reply:
Resolved in https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/134/commits/cb280258531486112852c33ef47efcaf2ccd4177.